source/github-helpers/index.ts

```diff
@@ -16,6 +16,16 @@ export const getConversationNumber = (): string | undefined => {
 	return undefined;
 };
 
+export function getCurrentBranchFromFeed(): string {
```

Comment:
Don't we have this function elsewhere? If not, did we drop it and why?

Reply:
See description of part 3

Reply:
I don’t think that answers my question 😜 

I found this:

- it was dropped in https://github.com/refined-github/refined-github/pull/4193, seemingly without a specific reason (other than not being useful anymore)

So restoring should be fine.  👍 

I'm hesitant to modify this function because we touched it _a lot_ over the years.

Some possible safety improvements:

- move the isRepoCommitList check to this function

Reply:
Nothing, because it does not know it. Void | string